client-src/index.js

```diff
@@ -15,7 +15,7 @@ import createSocketURL from "./utils/createSocketURL.js";
  * @property {boolean} hot
  * @property {boolean} liveReload
  * @property {boolean} progress
- * @property {boolean | { warnings?: boolean, errors?: boolean }} overlay
+ * @property {boolean | { warnings?: boolean, errors?: boolean, policyName?: string }} overlay
```

Comment:
I am wondering whether the `policyName` should be renamed to more verbose `trustedTypesPolicyName`. WDYT?

Reply:
I don't think we will have the `policyName` option here, but I think make sense to rename it to `trustedTypesPolicyName`, to avoid misleading

Reply:
Fixed in https://github.com/webpack/webpack-dev-server/pull/4404/commits/c63489b3372539de2b51fffa738249435f5ff6bc